modules/local/plink2_relabelpvar.nf

```diff
@@ -49,11 +49,11 @@ process PLINK2_RELABELPVAR {
         $set_ma_missing \\
         --pfile ${geno.baseName} $compressed \\
         --make-just-pvar zs \\
-        --out ${params.target_build}_${prefix}${meta.chrom}
+        --out ${meta.build}_${prefix}${meta.chrom}
 
     # cross platform (mac, linux) method of preserving symlinks
-    cp -a $geno ${params.target_build}_${prefix}${meta.chrom}.pgen
-    cp -a $pheno ${params.target_build}_${prefix}${meta.chrom}.psam
+    cp -a $geno ${meta.build}_${prefix}${meta.chrom}.pgen
```

Comment:
@nebfield should it not be `cp -a $geno ${meta.build}_${prefix}_${meta.chrom}.pgen` an underscore before chromo?

Reply:
`$prefix` contains a trailing underscore, but I'll change it because it's confusing 😄

Reply:
Not required, especially if it will break tons of modules!

Reply:
I had to update `PLINK2_RELABELPVAR` to get the `ANCESTRY_BOOTSTRAP` subworkflow working with the new king file schema, so I updated the other modules to be consistent 🤠